jenkins-job-generator/definitions/linux-package-jobs.yml

```diff
@@ -167,38 +166,44 @@
             }}
         }}
 
-- job:
-    name: recreate-debian-repository
-    description: 'Recreate debian repository.'
+- project:
+    name: resign-debian-repository
+    platform: *platforms
+    jobs:
+        - resign-debian-repository-{platform}
+
+- job-template:
+    name: resign-debian-repository-{platform}
+    description: 'Resign debian repository on {platform}.'
```

Comment:
Here should be an entry: `defaults: 'linux_package_global'`, like in the other template above.

Reply:
done